assets/js/app.js

```diff
@@ -23,17 +23,16 @@ const TAB_KEYCODE = 9;
 
 let Hooks = {};
 Hooks.CommandInput = {
+  mounted() {
+    const el = this.el;
+    this.handleEvent('reset-input', () => {
```

Comment:
Super minor thing, but wondering if calling the event just `reset` could be enough given the context of the event is already the "command input"?

Reply:
Yes sounds good